config/settings.local .yml.example

```
@@ -0,0 +1,5 @@
+---
+foreman:
+  token: foo
+  user: bar
```

```suggestion
  username: bar
```

I think this would be more consistent with the Foreman application.